module/aviatrix-controller-build/variables.tf

```diff
@@ -0,0 +1,177 @@
+  description = "Aviatrix Controller Version available in the Marketplace."
+  default     = "6.3.0"
+}
+
+variable "use_existing_network" {
+  type        = bool
+  description = "Use exist network."
+  default     = false
+}
+
+variable "vcn_cidr_block" {
+  type        = string
+  description = "VCN CIDR block."
+  default     = "10.0.0.0/16"
+}
+
+variable "vcn_dns_label" {
+  type        = string
+  description = "VCN DNS label."
+  default     = "vcn"
+}
+
+variable "vcn_display_name" {
+  type        = string
+  description = "VCN display name."
+  default     = "vcn"
+}
+
+variable "igw_display_name" {
+  type        = string
+  description = "IGW display name."
+  default     = "internet-gateway"
+}
+
+variable "vcn_id" {
+  type        = string
+  description = "VCN id."
+  default     = ""
+}
+
+variable "subnet_cidr_block" {
+  type        = string
+  description = "Subnet CIDR block."
+  default     = "10.0.0.0/24"
+}
+
+variable "subnet_display_name" {
+  type        = string
+  description = "Subnet display name."
+  default     = "subnet"
+}
+
+variable "subnet_dns_label" {
+  type        = string
+  description = "Subnet DNS label."
+  default     = "subnet"
+}
+
+variable "nsg_display_name" {
+  type        = string
+  description = "Network security group display name."
+  default     = "controller-sec-group"
+}
+
+# Standard OCI Variables for Compute and Networking Resources
+variable "availability_domain" {
+  type        = number
+  default     = 1
+  description = "OCI Availability Domains: 1,2,3  (subject to region availability)"
+}
+
+variable "vm_display_name" {
+  type        = string
+  description = "VM display name."
+  default     = "controller"
+}
+
+variable "vm_compute_shape" {
+  type        = string
+  description = "VM compute shape."
+  default     = "VM.Standard2.2" //2 cores
+}
+
+variable "subnet_id" {
+  type        = string
+  description = "Subnet id."
+  default     = ""
+}
+
+# Variables required for launching Aviatrix Controller
+variable "compartment_ocid" {
+  type        = string
+  description = "Compartment where infrastructure resources will be created"
+}
+
+variable "tenancy_ocid" {
+  type        = string
+  description = "Tenancy OCID."
+}
+
+variable "user_ocid" {
+  type        = string
+  description = "User OCID."
+}
+
+variable "license_model" {
+  type        = string
+  description = "Marketplace license model: 'BYOL' or 'PAID'."
+}
+
+variable "fingerprint" {
+  type        = string
+  description = "Fingerprint of the public key."
+}
+
+variable "incoming_ssl_cidrs" {
+  type        = list(string)
+  description = "The CIDRs to be allowed for HTTPS(port 443) access to the Aviatrix Controller"
+}
+
+variable "private_key_path" {
+  type        = string
+  description = "Private key file path."
+}
+
+variable "region" {
+  type        = string
+  description = "Region."
+}
+
+variable "ssh_public_key" {
+  type        = string
+  description = "Public key file path."
+}
```

Comment:
We usually don't enable SSH customers, and by the way, is SSH really enabled in the module?

Reply:
> We usually don't enable SSH customers, and by the way, is SSH really enabled in the module?

yes, I think currently the ssh_public_key is not used, but before the oci module request the ssh_public_key variable, so I keep it, should we need to remove this variable?

Reply:
> > We usually don't enable SSH customers, and by the way, is SSH really enabled in the module?
> 
> yes, I think currently the ssh_public_key is not used, but before the oci module request the ssh_public_key variable, so I keep it, should we need to remove this variable?

If a variable is not used, then remove it.